.travis.yml

```diff
@@ -61,6 +62,9 @@ before_install:
     mkdir -p $HOME/bin && cp ./aws-iam-authenticator $HOME/bin/aws-iam-authenticator && export PATH=$PATH:$HOME/bin
     export PATH=$PATH:$HOME/bin
     aws eks update-kubeconfig --name appsTeam --region=eu-west-1
+    sudo apt-add-repository ppa:qameta/allure
```

Comment:
Try adding the flag "-y" to the end of this command for avoiding the blocking message

Reply:
I omitted that:D
Thanks